packages/jellyfish-api-core/__tests__/category/masternode/resignMasternode.test.ts

```diff
@@ -0,0 +1,79 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasternodeState } from '../../../src/category/masternode'
+import { RpcApiError } from '../../../src'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should resignMasternode', async () => {
+    const ownerAddress = await client.wallet.getNewAddress()
+    const masternodeId = await client.masternode.createMasternode(ownerAddress)
```

Comment:
other than `resignMasterNode`, use container.call

Reply:
Yes, i tried using the `container.call` to get addresses but unfortunately it was failing. @jingyi2811  could also comment on that. 
Here's the error i get when i use `container.call`

<img width="788" alt="image" src="https://user-images.githubusercontent.com/57817584/123369893-92c25a80-d576-11eb-8b52-efda33dd5ce8.png">

Reply:
`container.getNewAddress` works fine. I will be using it.

Reply:
you need specify the address type

Reply:
Okay, did not know that. Thanks. 
I have already changed it to container.getMewAdress. Should i still use the container.call?